### DISCUSSION TITLE
How to ignore private key error?

### DISCUSSION THREAD

**Discussion by vegardhw** [secretlint/secretlint#295] [Q&A]
## Relevant information

How do you ignore the error reporting of a private key that is found in the commit history of a GitHub repository? The key is deleted and not in use but the error is still there because of the commit history.

Error is flagged during CI checks on a GitHub pull request. Error is not reported during manual run of secretlint on local cloned repository.

### Environment (if relevant)

secretlint v5.2.3

**Comment by azu** [comment]
secretlint does not lint for commit history by default #34 #211 

Something is wrong.

**Comment by vegardhw** [comment]
I should have mentioned that this is during CI checks in a PR on GitHub. Manual run of secretlint in local cloned copy of repository does not produce the same error.
